.github/workflows/compile.yml

```diff
@@ -19,66 +19,162 @@ jobs:
           sudo apt-get update -q
           sudo apt-get remove -y clang
           sudo apt-get install -y clang-10 build-essential autotools-dev autoconf-archive flex bison libjson-c-dev liblzo2-dev libglib2.0-dev libtool
-      - name: Get Xen version
-        id: get-xen-hash
+      - name: Get submodule hashes version
+        id: get-hash
         run: |
           echo ::set-output name=XEN_HASH::$(git submodule | grep xen | awk '{ print $1 }')
+          echo ::set-output name=LIBVMI_HASH::$(git submodule | grep libvmi | awk '{ print $1 }')
```

Comment:
What's the point of caching the LibVMI source files vs just checking it out form git?

Reply:
Umm, the cache stores all the build files too, so we can just use `make install` from the previously cached files and it will not rebuild the files again.

Reply:
Would make more sense to build the libvmi deb file and just store that

Reply:
Yeah, it is true. Could you please tell me how can that `deb` file be made?

Reply:
See https://github.com/libvmi/libvmi/blob/master/.github/workflows/make-deb.yml

Reply:
Done